generators/app/templates/src/config/api.js

```diff
@@ -25,7 +25,7 @@ export const apiSetup = dispatch => {
     if (response.status === STATUS_CODES.unauthorized) {
       /*
        * TODO: These callbacks should only be called if no other callback was asigned for the response.
-       * - dispatch(alertActions.error(i18next.t('apiErrors:expired')));
+       * - dispatch(alertActions.error(t('apiErrors:expired')));
```

Comment:
Where do we get `t` from here? 😅

Reply:
should be sent to `apiSetup`

Reply:
I don't think we should use dispatch from here. I'd delete this and if eventually someone wants to do this, they can pass a callback instead, so this setup doesn't depend on the way the error is called/saved